### ISSUE TITLE
NTR/OTR: Septin hourglass terms 

### ISSUE THREAD

**Issue by suzialeksander** [geneontology/go-ontology#19534]
I'm working on PMID: 32386534, about septin hourglass assembly/stability in Scer. I can't find many GO terms specifically about the hourglass, which appears to be a distinct form between GO:0005940 septin ring and GO:0032176 split rings (see Figure 1 of PMID: 21736496). 

1. OTR: It seems like https://www.ebi.ac.uk/QuickGO/term/GO:0032173 GO:0032173 septin collar should have a synonym of "septin hourglass" (see PMID: 32386534, where "collar" does not appear). 

2.  NTR: cellular bud neck septin hourglass organization: I found GO:0032186 cellular bud neck septin ring organization but since ring and collar are different terms, I think I need a new term for org.  Synonym "septin collar organization" as above. Not sure if this should be a child of GO:0032185 septin cytoskeleton organization or of  GO:0031106 septin ring organization

 3. NTR: septin collar assembly may also be needed to describe hourglass assembly, while the above organization is more about the transitional hourglass (PMID: 32386534  Graphical Abstract) and remodeling into a double ring (PMID: 32386534 Fig 4).  There's an existing  GO:0000921 septin ring assembly, it seems this NT would fit in a similar place and be a child of NTR septin hourglass organization

On that note, and out of curiosity, would  "GO:0031107 septin ring disassembly" apply to the disassembly of the structures that occur after the double rings form- the hourglass and then the split rings- or just the single ring?

**Comment by pgaudet** [comment]
@ValWood Do you have any insights on the questions above ?

**Comment by ValWood** [comment]
Not much , as  "septin collar"  to the bud neck they aren't applicable to fission yeast.

Septins are not very well studied in fission yeast compared to budding yeast, but we do have septin rings that split into double rings. I think the dissaembly term only applies to the split rings...(because the rings don't disassemble before they split)


v

**Comment by pgaudet** [comment]
Hi @suzialeksander 

Sorry about the delay - 

1. I've added  "septin hourglass" as EXACT synonym of GO:0032173 septin collar

2. I created the new term: 
+[Term]
+id: GO:0140544
+name: septin collar organization
+namespace: biological_process
+synonym: "A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of cytoskeletal structures comprising the septin collar." EXACT [PMID:21736496]
+synonym: "cellular bud neck septin hourglass organization" EXACT []
+intersection_of: GO:0016043 ! cellular component organization
+intersection_of: results_in_organization_of GO:0032173 ! septin collar
+property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/19534 xsd:anyURI
+created_by: pg
+creation_date: 2020-11-10T13:47:48Z

- However I flipped the label and synonym with respect to what you requested, to match the existing term GO:0032173 septin collar

- WRT parents, the logical definition classifies this term as 'GO:0032185 septin cytoskeleton organization' so I suggest we go with that. 

3. 
> would "GO:0031107 septin ring disassembly" apply to the disassembly of the structures that occur after the double rings form- the hourglass and then the split rings- or just the single ring?

The definition doesn't specify this. Should we be more clear about start and end ? 

Thanks, Pascale
